src/lib/application.py

```diff
@@ -537,6 +538,31 @@ def open_report_page(self):
         self.click_element(locator_type="image", locator="open_report_button.png")
         self.wait_find_element(locator_type="image", locator="report_page_header.png")
 
+    def verify_story_points(self, target):
+        """
+        Verifies the story points in report after analysis
+
+        Returns:
+            (bool): True if story points were accurate
+        """
+        if target == "eap7":
+            story_point_locator = "eap7_story_points.png"
```

Comment:
Report being html , can we not read the number from the UI ?

Reply:
@sshveta , that can be done. We'd have to use Selenium/webdriver to access web page elements . I'll add support for that in a future PR.